napari/conftest.py

```diff
@@ -332,3 +332,28 @@ def pytest_collection_modifyitems(session, config, items):
             at_end.append(items.pop(i))
 
     items.extend([x for _, x in sorted(zip(put_at_end, at_end))])
+
+
+@pytest.fixture
+def test_napari_plugin_manager(monkeypatch):
```

Comment:
Very nice! Is this something we could make more easy use of for our plugin dev tools stuff? it's not clear that an individual plugin dev would need to use it, but we can reuse it elsewhere most likely (cc @ziyangczi @neuromusic)

Reply:
yeah, for sure.  I stopped short of putting it in `_testsupport` for now... but it is certainly intended as something that should be isolated from other stuff, for exactly that kind of purpose.
the one thing we might want to do there though is _not_ call `pm._initialize` (leaving out the initialization of our builtin stuff)... leaving it to the tests that want our stuff to do that manually.  (to answer your other question, that was why `_initialize` wasn't originally called in init)